routers/commentsRouter_controllers.go

```diff
@@ -0,0 +1,28 @@
+package routers
+
+import (
+	beego "github.com/beego/beego/v2/server/web"
```

Comment:
@junan But this file, I'll keep like this because of this file is auto generated.